src/pages/academy/components/_social-sharing.js

```diff
@@ -28,61 +29,185 @@ const StyledFlex = styled(Flex)`
     }
 `
 
+const SharingWrapper = styled.div`
+    display: flex;
+    position: absolute;
+    left: 73%;
+    top: 25px;
+
+    img {
+        width: 21px;
+        height: 23px;
+    }
+
+    @media ${device.tabletL} {
+        left: 85%;
+        top: 8px;
+    }
+`
+
+const SharingButton = styled.button`
+    position: absolute;
+    width: 32px;
+    height: 32px;
+    left: 0;
+    top: 5px;
+    background: #ffffff;
+    border: 0 solid rgba(236, 241, 247, 0.5);
+    border-radius: 4px;
+
+    :focus-within {
+        background: #f2f3f4;
+    }
+    img {
+        width: 21px;
+        height: 23px;
+        position: absolute;
+        top: -10%;
+        left: -10%;
+    }
+`
+
+const DropDownSharingButton = styled.div`
+    position: absolute;
+    left: -5px;
+    top: 50px;
+    width: 40px;
+    background-color: #ffffff;
+    border: 1px solid #ffffff;
+    box-shadow: 0 16px 24px 2px rgba(0, 0, 0, 0.14);
+
+    &::after {
+        border: 5px solid transparent;
+        border-bottom: 10px solid white;
+        content: '';
+        position: absolute;
+        right: 13px;
+        top: -15px;
+        z-index: 2;
+    }
+`
+
+export const SocialWrapper = styled.div`
+    grid-area: social;
+
+    img {
+        margin-right: 2rem;
+    }
+
+    @media ${device.tabletL} {
+        display: flex;
+        justify-content: center;
+        margin: 3rem 0 1rem;
+
+        img {
+            margin: 1rem 1rem 1rem 1rem;
+        }
+    }
+`
+
+// clip-path: polygon(50% 60%, 20% 100%, 80% 100%);
+const url = typeof window !== 'undefined' ? window.location.href : ''
+const social_media = [
```

Comment:
Let's create an utility function with this, later import into this component

Reply:
Did this. Thanks for advice